apps/web/src/actions/users/fetch.ts

```diff
@@ -10,7 +10,7 @@ export const getUsersActions = authProcedure
   .handler(async ({ ctx }) => {
     const usersScope = new UsersRepository(ctx.workspace.id)
 
-    return await usersScope
```

Comment:
remove the async declaration then also

Reply:
I'll do but does not matter no? This method keeps being async